### DISCUSSION TITLE
[Shell] Change Brightness With Twinkle Tray

### DISCUSSION THREAD

**Discussion by sapphired18** [AutoDarkMode/Windows-Auto-Night-Mode#750] [Custom scripts]
## Twinkle Tray should run at the startup and/or in the background to use this script

```
  - Name: Twinkle-Tray
    Command: C:\Users\user\AppData\Local\Programs\twinkle-tray\Twinkle Tray.exe
    ArgsLight: [--MonitorNum=%, --Set=%, --Overlay]
    ArgsDark: [--MonitorNum=%, --Set=%, --Overlay]
    AllowedSources: [Any]
```
### Supported args:

- `--List` List all displays. *(available in v1.14.0+)*
- `--MonitorNum` Select monitor by number. Starts at 1. *Example: `--MonitorNum=2`*
- `--MonitorID` Select monitor by internal ID. Partial or whole matches accepted. *Example: `--MonitorID="UID2353"`*
- `--All` Flag to select all monitors.
- `--Set` Set brightness percentage. *Example: `--Set=95`*
- `--Offset` Adjust brightness percentage. *Example: `--Offset=-20`*
- `--VCP` Send a specific DDC/CI VCP code and value instead of brightness. The first part is the VCP code (decimal or hexadecimal), and the second is the value. *Example: `--VCP="0xD6:5"`* *(available in v1.14.4+)*
- `--Overlay` Flag to show new brightness levels in the overlay *Example: `--Overlay`*
- `--Panel` Flag to show new brightness levels in the panel *Example: `--Panel`*

*This feature is not available on the Windows Store version of Twinkle Tray.*

**Comment by JulienMaille** [comment]
I just discovered Twinkle Tray thanks to you.
Are you sure it won't work with the Windows store version provided you use the right filepath?

**Reply by Spiritreader** [reply]
Oh sorry I forgot to tag this!

Thank you for the script!
